west.yml

```diff
@@ -199,7 +199,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 0ea9c65e8e14fe01c055e571ad2e6fe5337dc58d
+      revision: d291bdcc4a59bace5ae7453e777e06080ccda8ce
```

Comment:
What's this? Doesn't seem related to any language fixes.

Reply:
idk where this stemmed from. I didn't change anything in this file.

I did this to squash commits. Maybe this squashed a different commit.

`git fetch
git reset --soft HEAD~2
git commit
git push -f`

Reply:
I think this is too much effort for you folks to have so many reviews for such a small change. I will close this PR, raise a new one in draft mode and make sure it passes the pipeline before raising it for your review. Thanks for your time.

Reply:
You can drop that commit and then the PR is good.

Reply:
@baaatli when you have just a single commit, you can update it simply using `git commit --amend`. You can always check the content what you're about to push using e.g. `git show` (for just a single commit) or e.g. `git log -p upstream/main..`. Also, no need to close this PR and create a new one. Just fix up your local tree, and once you've verified that everything is fine do another `git push --force`.